### ISSUE TITLE
Transition to supporting JAWS with Chrome rather than Firefox

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/QA#217]
Due to the performance of Firefox Quantum with JAWS 2018, we would like to transition to support JAWS with Chrome because Chrome is much much faster with JAWS. We should spot check JAWS with Chrome to make sure that there aren't too many odd issues on that platform. Then, if all seems OK, we should update the QA documentation to reflect the change.

**Comment by jessegreenberg** [comment]
Here are the sims I would like to test with Latest chrome and JAWS:
- [ ] john-travoltage
- [ ] balloons-and-static-electricity
- [ ] resistance-in-a-wire

I am going to do some testing on my machine to see how it all sounds on Chrome, but it would be very helpful to see what others find too. @ariel-phet could you please assign priority to this issue?

**Comment by jessegreenberg** [comment]
Notes from @jessegreenberg testing (I will edit this comment with things I find):
- I noticed that when reading with the virtual cursor PhET's focus highlight doesn't appear. But once an element is clicked it appears. But I found that this is happening in FF now too. So by default JAWS is no longer synchronizing DOM focus with virtual focus when reading.

**Comment by jessegreenberg** [comment]
In https://github.com/phetsims/a11y-research/issues/105, it was mentioned that Chrome does not (and probably will never) support MathML, if that impacts the decision here.

But the positive experience I am seeing in https://github.com/phetsims/QA/issues/217#issuecomment-436455512 makes me think that we are much better off supporting Chrome.

**Comment by jessegreenberg** [comment]
I feel confident now that this is the correct transition. 

But over slack, @KatieWoe said
>  I know I accidentally tested with chrome instead of firefox once and saw some pretty odd issues.

@KatieWoe if you have any specifics could you please list in this issue? No worries if you don't remember. It would still be helpful for someone to double check the list in https://github.com/phetsims/QA/issues/217#issuecomment-436455512 if there is time.

While testing, JAWS released version 2019 last month so please make sure that JAWS is up to date on the versions that are being tested.

Removing my assignment.

**Comment by KatieWoe** [comment]
@jessegreenberg If I recall correctly, It appeared that text was being highlighted as the scene description was read.  No such text was visible (which is good).  I may be remembering NVDA instead of Jaws though.  I've been doing some other testing, so I don't know if it is happening with Jaws.  There might have been a few other things that I don't remember.

**Comment by ariel-phet** [comment]
@phet-steele is this some testing you are also available for?

**Comment by jessegreenberg** [comment]
We already got good testing in https://github.com/phetsims/QA/issues/219, and I don't think we will get much more out of spending time on john-travoltage because teh a11y components are so similar to RIAW. We are going to proceed with supporting Chrome with JAWS.

**Comment by jessegreenberg** [comment]
@KatieWoe I updated places in QA documentation that refer to this in the above commits. The only other place that I could find was tester-book.tex, does that need to be updated too? Looks like almost same content as the qa-book.md so I was wondering that was generated or something. Is there anything else we need to document for this change?

**Comment by KatieWoe** [comment]
That document is the old version of the QA book that would be turned in to a pdf.  It does not need to be updated as we are switching to the version that you updated.  The issue templates would also need to be updated, as I believe they list the supported platforms.  I'm not sure though, so I can make the checks and updates there myself.  Thanks @jessegreenberg

**Comment by jessegreenberg** [comment]
OK, thanks @KatieWoe! In that case I think this issue can be closed.
